### PR TITLE
System data storage architectures update

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md [hunk 1]
```diff
@@ -4,52 +4,62 @@ uid: Supported_system_data_storage_architectures
...
+> [!NOTE]
+> Using managed services from a cloud provider instead of on-premises databases is supported from DataMiner 10.3.0/10.3.3 onwards. Using an on-premises OpenSearch database is also supported from DataMiner 10.3.0/10.3.3 onwards.
 
 ### Single DMA setups
 
-In a development environment with limited load, it is possible to host DataMiner, Cassandra, and Elasticsearch on one Windows machine. In this case, Elasticsearch and DataMiner must be installed on a separate disk or partition. However, note that this is not recommended for normal production environments.
+We recommend running DataMiner, Cassandra, and Elasticsearch/OpenSearch on **dedicated machines**, or using **managed services from a cloud provider**. At present, we support the Amazon Keyspaces Service and Amazon OpenSearch Service. DataMiner currently still needs to run on premises, but we intend to soon make it possible to deploy a DataMiner node as a service.
```
Comment (RobbeDGH) [inline user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md:31]: _DataMiner currently still needs to run on premises_

Maybe reword this to _still needs to be managed_?
Technically you can host your dma not on prem

File: user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md [hunk 2]
```diff
@@ -4,52 +4,62 @@ uid: Supported_system_data_storage_architectures
...
+In a development environment with **limited load**, it is possible to host DataMiner, Cassandra, and Elasticsearch on **one Windows machine**. In this case, Elasticsearch and DataMiner must be installed on a separate disk or partition. However, note that this is not recommended for normal production environments.
+
+![Development setup: DataMiner, Cassandra, and Elasticsearch hosted on the same machine](~/user-guide/images/Development-setup-DataMiner-Cassandra-and-Elasticsearch.png)<br>
+*Development setup: DataMiner, Cassandra, and Elasticsearch hosted on the same machine*
 
 ### Multiple DMA (non-Failover) setups
 
-In case you have more than one DataMiner Agent, you can scale on three levels: DataMiner, Cassandra, and Elasticsearch.
+In case you have more than one DataMiner Agent, you can scale both on DataMiner level and on database level. You can also use managed services from a cloud provider. At present, we support the Amazon Keyspaces Service and Amazon OpenSearch Service. DataMiner currently still needs to run on premises, but we intend to soon make it possible to deploy a DataMiner node as a service.
```
Comment (RobbeDGH) [inline user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md:50]: same

File: user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md [hunk 3]
```diff
@@ -59,19 +69,20 @@ For Cassandra, any number of nodes can be used, ideally running on Linux machine
...
-
-To be fully hardware-redundant, we recommend at least three machines hosting Elasticsearch nodes, two machines hosting the DataMiner Failover pair and two machines hosting Cassandra nodes. More Cassandra and Elasticsearch nodes can be needed depending on the load on your system. Check the [node calculator](https://community.dataminer.services/calculator/) for more information.
+An on-premises setup should at the very least consist of two DataMiner machines, one Elasticsearch machine, and one Cassandra machine. However, if the Cassandra or Elasticsearch machine are no longer available in this setup, you will no longer be able to start DataMiner, so this is not recommended for redundancy setups. To be fully hardware-redundant, we recommend **at least three machines hosting Elasticsearch** nodes, **two machines hosting the DataMiner Failover pair** and **two machines hosting Cassandra nodes**. More Cassandra and Elasticsearch nodes can be needed depending on the load on your system. Check the [**node calculator**](https://community.dataminer.services/calculator/) for more information.
```
Comment (RobbeDGH) [inline user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md:74]: Note that the node calculator page says its under development and is a preview but also says the latest updated was the first of april last year. Is this up to date?

**Review by RobbeDGH — COMMENTED** [review]
The logo next to the elastic nodes in the images is using the logo of the old 'Amazon Elasticsearch Service' which is deprecated and is replaced by the 'Amazon OpenSearch Service' (which offers both Opensearch as well as elasticsearch) and has a new logo.
I believe this one is up to date:
image

Furthermore, maybe its possible to change the elastic logos to be a combination of elastic with opensearch? Like a circle thats half the logo of elasticsearch and half the logo of opensearch. They are both circles so might be possible. 
Just an idea though.

**Comment by RobbeDGH** [comment]
> The logo next to the elastic nodes in the images is using the logo of the old 'Amazon Elasticsearch Service' which is deprecated and is replaced by the 'Amazon OpenSearch Service' (which offers both Opensearch as well as elasticsearch) and has a new logo. I believe this one is up to date: image
> 
> Furthermore, maybe its possible to change the elastic logos to be a combination of elastic with opensearch? Like a circle thats half the logo of elasticsearch and half the logo of opensearch. They are both circles so might be possible. Just an idea though.

This is what i mean (excuse my poor paint job)

image

**Comment by RobbeDGH** [comment]
> > The logo next to the elastic nodes in the images is using the logo of the old 'Amazon Elasticsearch Service' which is deprecated and is replaced by the 'Amazon OpenSearch Service' (which offers both Opensearch as well as elasticsearch) and has a new logo. I believe this one is up to date: image
> > Furthermore, maybe its possible to change the elastic logos to be a combination of elastic with opensearch? Like a circle thats half the logo of elasticsearch and half the logo of opensearch. They are both circles so might be possible. Just an idea though.
> 
> This is what i mean (excuse my poor paint job)
> 
> image

(but then of course line up the cirlces)

**Comment by RobbeDGH** [comment]
instead of node calculator, maybe this page can be used instead:
https://docs.dataminer.services/user-guide/Advanced_Functionality/Databases/Cassandra_database/Replication_and_consistency_configuration.html

File: user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md (hunk 3 above)
Comment (MariekeGO) [reply user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md:74]: @BertVandenberghe, what do you think? Can we refer to the node calculator? Does that page still need the preview label?

**Comment by MariekeGO** [comment]
> The logo next to the elastic nodes in the images is using the logo of the old 'Amazon Elasticsearch Service' which is deprecated and is replaced by the 'Amazon OpenSearch Service' (which offers both Opensearch as well as elasticsearch) and has a new logo. I believe this one is up to date: image
> 
> Furthermore, maybe its possible to change the elastic logos to be a combination of elastic with opensearch? Like a circle thats half the logo of elasticsearch and half the logo of opensearch. They are both circles so might be possible. Just an idea though.

Edith has provided new images. In a later stage, we can have these redesigned by Marketing, but for now these will hopefully suffice to make everything clear.

File: user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md (hunk 3 above)
Comment (BertVandenberghe) [reply user-guide/Advanced_Functionality/Databases/Supported_System_Data_Storage_Architectures.md:74]: Yes, you can definitely refer to the node calculator. As things are still evolving and being evaluated, the preview label will still remain for the time being.
